src/lib.rs

```diff
@@ -664,13 +660,11 @@ impl<T> Webauthn<T> {
     {
         let chal = self.generate_challenge();
 
-        let verified = creds.iter()
-            .all(|cred| cred.verified);
-        let unverified = creds.iter()
-            .all(|cred| !cred.verified);
+        let verified = creds.iter().all(|cred| cred.verified);
```

Comment:
Did you manually change these or did rust fmt do it?

Reply:
This was `cargo fmt`

Reply:
No problem then was curious is all. What version of rust are you using? I've seen some issues with cargo fmt in the past where different versions output different code and so if people have different versions they end up fighting over the formatting ....

Reply:
The active toolchain section from `rustup show` reads:
```
stable-x86_64-apple-darwin (default)
rustc 1.47.0 (18bf6b4f0 2020-10-07)
```

Reply:
Cool no problems, that explains it. I'm still using 1.45.2.